pandas/tests/io/parser/test_common.py

```diff
@@ -2160,6 +2160,14 @@ def test_suppress_error_output(all_parsers, capsys):
     assert captured.err == ""
 
 
+def test_err_msg_bytes():
+    # GH#29233
+    try:
+        pd.read_csv("nonexistent_name")
+    except FileNotFoundError as err:
+        assert err.strerror == "File nonexistent_name does not exist"
```

Comment:
can pytest.raises not be used?

Reply:
good idea.  i think this was split off from a branch where i was also checking err.errno

Reply:
also looks like theres another test i can merge this into